src/tron/tronInfo.ts

```diff
@@ -1,8 +1,88 @@
-import { EdgeCurrencyInfo } from 'edge-core-js/types'
+import { EdgeCurrencyInfo, EdgeTokenMap } from 'edge-core-js/types'
 
 import { makeOuterPlugin } from '../common/innerPlugin'
+import { makeMetaTokens } from '../common/tokenHelpers'
 import type { TronTools } from './tronPlugin'
-import { TronNetworkInfo } from './tronTypes'
+import type { TronNetworkInfo } from './tronTypes'
+
+const builtinTokens: EdgeTokenMap = {
```

Comment:
The previous code flow is a bit tough to follow, but wouldn't Tron tokens also go through the `loadInnerPlugin()` routine which calls `upgradeMetaTokens` which would have lowercased the tokenIds? This is what I ran into when implementing XRP even though my getTokenId didn't do the lowercasing. While the true upper/lower case tokenIds would have been preferred for Tron, we might have tokenIds already saved in user's accounts that are all lowercase.

Reply:
After a sidebar with Matthew to discuss pros & cons, we have decided to go ahead and change the tokenId's. We have a plan for dealing with the fallout, which isn't as terrible as we thought at first.